auto/src/glew.gperf

```diff
@@ -0,0 +1,674 @@
+%struct-type
```

Comment:
Generally speaking, generated files do not typically get committed to the repository.